dist/client.js

```diff
@@ -459,7 +459,6 @@
     define([], function() {
       return CrossStorageClient;
     });
-  } else {
-    root.CrossStorageClient = CrossStorageClient;
   }
+  root.CrossStorageClient = CrossStorageClient;
```

Comment:
That's not an issue with this library, but the electron setup. If cross-storage is installed under node_modules, or correctly in module.paths, then you don't need to import with `<script src="../.../client.js"></script>` For clientside access, you can simply `require('cross-storage').CrossStorageClient`